core/kubernetes/main/hapi-fhir/hapi-fhir-server-deployment.yaml

```diff
@@ -23,6 +23,9 @@ spec:
       containers:
         - name: hapi-fhir-server
           image: hapiproject/hapi:latest
+          resources:
+            limits:
+              memory: "2Gi"
```

Comment:
Please implement the memory limit here the same way as in Docker (Env Vars)
We want the installations to be a similar as possible